android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/contract/FormsViewContract.java

```diff
@@ -0,0 +1,27 @@
+package com.liferay.mobile.formsscreenletdemo.contract;
+
+import com.liferay.mobile.formsscreenletdemo.view.FormsActivity;
+import kotlin.Unit;
+
+/**
+ * @author Luísa Lima
+ */
+public interface FormsViewContract {
+	interface FormsView {
+
+		void hideProgress();
+
+		void recyclerViewWorkaround();
+
+		Unit showError(String message);
+	}
+
+	interface FormsPresenter {
+
+		void onActivityCreated(FormsActivity formsActivity);
```

Comment:
Would be nice if you'd create a IBasePresenter (using generics) to abstract onActivityCreated and onActivityDestroyed. What do you think?